common/test/acceptance/pages/lms/video/video.py

```diff
@@ -302,8 +317,20 @@ def is_captions_visible(self):
 
         """
         self.wait_for_ajax()
-        caption_state_selector = self.get_element_selector(CSS_CLASS_NAMES['closed_captions'])
-        return not self.q(css=caption_state_selector).present
+        caption_state_selector = self.get_element_selector(CSS_CLASS_NAMES['captions'])
+        return self.q(css=caption_state_selector).visible
+
+    def is_closed_captions_visible(self):
+        """
+        Get current visibility sate of closed captions.
+
+        Returns:
+            bool: True means captions are visible, False means captions are not visible
+
+        """
+        self.wait_for_ajax()
+        closed_caption_state_selector = self.get_element_selector(CSS_CLASS_NAMES['closed_captions'])
+        return not self.q(css=closed_caption_state_selector).invisible
```

Comment:
Use the `self.q(...).visible` property instead of `not self.q(...).invisible`.   The way this is written, it is possible that `is_closed_captions_visible` could return true when the element is actually not present.  (https://github.com/edx/bok-choy/blob/master/bok_choy/query.py#L439-L446)